### ISSUE TITLE
Missing pre-processing data?

### ISSUE THREAD

**Issue by elun15** [yrims/AIC19#2]
Hello, 

Is there any pre-processing of the raw data given by the challenge missing in the repo?

Please, could you provide information about how you organize/process the video sequences of the challenge in order to run the code? 

If we download the dataset from the link you provide, it is not in the format S1c01 ~ S5c36.  (as it is required in the readme file and in the code). Do you create S1c01 ~ S5c36 folders manually?

Thank you, 

Elena

**Comment by imranzaman5202** [comment]
yes you have to create the names of the folders manually in dataset/data directory

**Comment by guangqianqin** [comment]
it have a error when i run get_track_info.py ,because the calibration.txt is different in test or train documents
